test/jest/ui/nFormat.test.js

```diff
@@ -0,0 +1,248 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why is this needed?

Reply:
Because of the unused import of `jest`. (Copied from DynamicRamCalculation.test.js, not sure if I can remove it without consequences)